client/src/app/forms/rsvp/rsvp.component.ts

```diff
@@ -10,7 +10,7 @@ import { RecursivePartial } from '@libs/utils';
   styleUrls: ['./rsvp.component.scss']
 })
 export class RsvpFormComponent {
-	@Output() submit = new EventEmitter<RecursivePartial<rsvpData>>()
+	@Output() submit = new EventEmitter<RecursivePartial<primaryData & contactData & rsvpData>>()
```

Comment:
Don't be mad - this might be a thing we want to have as a separate type. This is..a little gross.